README.md

```diff
@@ -148,83 +78,115 @@ ray up deploy/ray/aviary-cluster.yaml
 ray attach deploy/ray/aviary-cluster.yaml
 
 # Deploy the LightGPT model. 
-aviary run --model ~/models/continuous_batching/amazon--LightGPT.yaml
+cd aviary && serve run serve/amazon--LightGPT.yaml
```

Comment:
why's cd aviary required?

Reply:
`serve run` needs to be call from the repo's root directory. We used `cd aviary` to express that.

I'm not sure if it's needed though. I think using the full file path (`serve/...`) should be enough. What do you think?